tests/testsuite/owner.rs

```diff
@@ -91,6 +94,40 @@ Caused by:
         .run();
 }
 
+#[cargo_test]
+fn simple_remove() {
```

Comment:
nit: for review and `git blame` purposes, its nice to not move functions along with functionality changes but to leave those to their own commit

Reply:
OK, I'll pay attention. thanks for reminding.